#! /usr/bin/env python

PACKAGE='jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("offset_x", int_t, 0, "x coordinates of top-left point", 0, 0, 5096)
gen.add("offset_y", int_t, 0, "y coordinates of top-left point", 0, 0, 5096)
gen.add("width", int_t, 0, "width", 256, 0, 5096)
gen.add("height", int_t, 0, "height", 256, 0, 5096)

exit(gen.generate(PACKAGE, "jsk_perception", "MaskImageGenerator"))
